security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -935,6 +936,12 @@
                         <sha1>SHA1</sha1>
                     </OptionValues>
                 </dns_kas_authtype>
+                <dns_dedyn_token type="TextField">
```

Comment:
These fields should use "dns_desec" as well :)

Reply:
uff. you're so right. thanks for your patience!